entity-framework/core/providers/index.md

```diff
@@ -20,11 +20,11 @@ Entity Framework Core can access many different databases through plug-in librar
-| [Microsoft.EntityFrameworkCore.Sqlite](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.Sqlite)                                                                           | SQLite 3.7 onwards              | [EF Core Project](https://github.com/dotnet/efcore/) (Microsoft)                                |                                            | 3.1               | [docs](xref:core/providers/sqlite/index)                                                                                                       |
-| [Microsoft.EntityFrameworkCore.InMemory](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.InMemory)                                                                       | EF Core in-memory database      | [EF Core Project](https://github.com/dotnet/efcore/) (Microsoft)                                | [Limitations](xref:core/testing/in-memory) | 3.1               | [docs](xref:core/providers/in-memory/index)                                                                                                    |
-| [Microsoft.EntityFrameworkCore.Cosmos](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.Cosmos)                                                                           | Azure Cosmos DB SQL API         | [EF Core Project](https://github.com/dotnet/efcore/) (Microsoft)                                |                                            | 3.1               | [docs](xref:core/providers/cosmos/index)                                                                                                       |
-| [Npgsql.EntityFrameworkCore.PostgreSQL](https://www.nuget.org/packages/Npgsql.EntityFrameworkCore.PostgreSQL)                                                                         | PostgreSQL                      | [Npgsql Development Team](https://github.com/npgsql)                                            |                                            | 3.1               | [docs](https://www.npgsql.org/efcore/index.html)                                                                                               |
+| [Microsoft.EntityFrameworkCore.SqlServer](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.SqlServer)                                                                     | SQL Server 2012 onwards         | [EF Core Project](https://github.com/dotnet/efcore/) (Microsoft)                                |                                            | 5.0               | [docs](xref:core/providers/sql-server/index)                                                                                                   |
```

Comment:
Should we update everything to 5.0 removing 3.1 or should we leave 3.1 in? as 3.1 is LTS.

Reply:
Hmm... Maybe we change the column title to "Latest supported version", and implicitly 5.0 means that 3.1 is also supported? Just to keep things simple and convey the important information.